openmdao/solvers/nl_gauss_seidel.py

```diff
@@ -137,6 +151,46 @@ def solve(self, params, unknowns, resids, system, metadata=None):
             normval = resids.norm()
             u_norm = np.linalg.norm(unknowns.vec - unknowns_cache)
 
+            if self.options['use_aitken']: # If Aitken acceleration is enabled
```

Comment:
Could you pull the latest master into your pull request, @shamsheersc19 ?
We did some fiddling with the pyoptsparse wheels, and if you pull that in, we can see if your PR passes now, which should get us closer to accepting it.